ContinueEycaActivation/index.ts

```diff
@@ -0,0 +1,80 @@
+﻿import { AzureFunction, Context } from "@azure/functions";
+import * as df from "durable-functions";
+import { toError } from "fp-ts/lib/Either";
+import { fromEither, tryCatch } from "fp-ts/lib/TaskEither";
+import * as t from "io-ts";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+import { OrchestratorInput } from "../StartEycaActivationOrchestrator/index";
+import { trackException } from "../utils/appinsights";
+import { Failure, PermanentFailure, TransientFailure } from "../utils/errors";
+import { makeEycaActivationOrchestratorId } from "../utils/orchestrators";
+
+export const ContinueEycaActivationInput = t.type({
+  fiscalCode: FiscalCode
+});
+export type ContinueEycaActivationInput = t.TypeOf<
+  typeof ContinueEycaActivationInput
+>;
+
+const permanentDecodeFailure = (errs: t.Errors) =>
+  Failure.encode({
+    kind: "PERMANENT",
+    reason: `Cannot decode input: ${readableReport(errs)}`
+  });
+
+const transientOrchestratorError = (err: unknown) =>
+  Failure.encode({
+    kind: "TRANSIENT",
+    reason: `Error starting the orchestrator: ${toError(err).message}`
+  });
+
+/**
+ * Reads from a queue the fiscalCode
+ * then try to start the orchestrator to activate the EYCA card.
+ */
+export const index: AzureFunction = (
+  context: Context,
+  message: unknown
+): Promise<Failure | string> => {
+  return fromEither(ContinueEycaActivationInput.decode(message))
+    .mapLeft(permanentDecodeFailure)
+    .chain(({ fiscalCode }) =>
+      tryCatch(
+        () =>
+          df.getClient(context).startNew(
```

Comment:
Why we need an orchestrator instead of doing the operation inside this Activity?

Reply:
Because there are others operations that need retry logic (I will add them later with other PRs) :)